package/gluon-status-page/javascript/status-page.js

```diff
@@ -39,6 +39,10 @@
 	function prettyPrefix(prefixes, step, d) {
 		var prefix = 0;
 
+		if (d == undefined) {
+			return "-"
```

Comment:
The returned value should be `"- "`, so there is a space between the `-` and the unit.

Reply:
I thought about that. returning "-" here should lead to no unit at all,  am I reading this wrong?

Reply:
prettyBits and prettyBytes will still add "bps"/"B".